### PR TITLE
Introduce Strict Output Seam TOD

### PR DESCRIPTION
**Pull request by seveibar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,3 +72,19 @@ The following CANNOT be converted to `seam-tod` strings:
...
+is exactly one Strict Seam TOD Output representation of a timezone time pair. The
+rules for constructing a Strict Seam TOD Output are as follows:
+
+- Start with `HH:MM:SS.mmm[TIMEZONE]`
+  - `HH` - hours with `0` left pad such that it is exactly two characters
+  - `MM` - minutes with `0` left pad such that it is exactly two characters
+  - `SS` - seconds with `0` left pad such that it is exactly two characters
+  - `mmm` - milliseconds with `0` left pad such that it is exactly three characters
+  - `TIMEZONE` - the relevant IANA timezone
+- If representing an offset, compute the offset by adjusting for a new `HH`, `MM`, `SS` and `mmm` and set the `TIMEZONE` to `UTC`
```
Comment (seveibar) [inline README.md]:
```suggestion
- If representing an offset (not a timezone), compute the offset by adjusting for a new `HH`, `MM`, `SS` and `mmm` and set the `TIMEZONE` to `UTC`
```

File: README.md [hunk 2]
```diff
@@ -72,3 +72,19 @@ The following CANNOT be converted to `seam-tod` strings:
...
+
+- Start with `HH:MM:SS.mmm[TIMEZONE]`
+  - `HH` - hours with `0` left pad such that it is exactly two characters
+  - `MM` - minutes with `0` left pad such that it is exactly two characters
+  - `SS` - seconds with `0` left pad such that it is exactly two characters
+  - `mmm` - milliseconds with `0` left pad such that it is exactly three characters
+  - `TIMEZONE` - the relevant IANA timezone
+- If representing an offset (not a timezone), compute the offset by adjusting for a new `HH`, `MM`, `SS` and `mmm` and set the `TIMEZONE` to `UTC`
+- If `mmm` is `000` and `SS` is `00`, remove `SS.mmm`
+- If `SS` is NOT `00` and `mmm` is `000`, remove `.mmm`
```
Comment (seveibar) [inline README.md:90]:
```suggestion
- If `SS` is NOT `00` and `mmm` is `000`, remove `.mmm`


### Examples

- `01:23:12.543[PST]` - 1:23am plus 12.543 seconds Pacific Standard Timezone
- `01:23:12[PST]` - 1:23am plus 12 seconds Pacific Standard Timezone
- `01:23[PST]` - 1:23am Pacific Standard Timezone
- `01:00[PST]` - 1am Pacific Standard Timezone
```

**Review by codetheweb — APPROVED** [review]
would be good to rfc this if we're going to start using it, not sure if many people have seen it/gave their thoughts

**Comment by seveibar** [comment]
@codetheweb yea will RFC it internally

**Pull request merged**